topics/tutorial-first-steps-with-kotlin-rpc.topic

```diff
@@ -0,0 +1,600 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE topic
+        SYSTEM "https://resources.jetbrains.com/writerside/1.0/xhtml-entities.dtd">
+<topic xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="https://resources.jetbrains.com/writerside/1.0/topic.v2.xsd"
+       title="First steps with Kotlin RPC" id="tutorial-first-steps-with-kotlin-rpc">
+
+    <show-structure for="chapter" depth="2"/>
+
+    <web-summary>
+        Discover the fundamentals of RPC and explore a detailed comparison between RPC and REST in this comprehensive
+        guide. Learn how to create your first application with Kotlin RPC.
+    </web-summary>
+    <link-summary>
+        Learn how to create your first application with Kotlin RPC and Ktor.
+    </link-summary>
+    <card-summary>
+        Learn how to create your first application with Kotlin RPC and Ktor.
+    </card-summary>
+    <tldr>
+        <p><b>Code examples</b>:
+            <a href="https://github.com/ktorio/ktor-documentation/tree/%ktor_version%/codeSnippets/snippets/tutorial-kotlin-rpc-client">
+                tutorial-kotlin-rpc-client
+            </a>,
+            <a href="https://github.com/ktorio/ktor-documentation/tree/%ktor_version%/codeSnippets/snippets/tutorial-kotlin-rpc-server">
+                tutorial-kotlin-rpc-server
+            </a>
+        </p>
+        <p>
+            <b>Used plugins</b>: <a href="server-routing.md">Routing</a>, <a href="client-websockets.md">WebSockets</a>,
+            <a href="https://kotlinlang.org/api/kotlinx.serialization/">kotlinx.serialization</a>,
+            <a href="https://github.com/Kotlin/kotlinx-rpc">kotlinx.rpc</a>
+        </p>
+    </tldr>
+    <p>
+        Kotlin RPC (Remote Procedure Call) is a new and exciting addition to the Kotlin ecosystem, which builds on well-established foundations
+        and runs on the <code>kotlinx.rpc</code> library.
+    </p>
+    <p>
+        The <code>kotlinx.rpc</code> library enables you to make procedure calls over network
+        boundaries using nothing but regular Kotlin language constructs. As such, it provides an alternative to both
+        REST and Google RPC (gPRC) while (in upcoming functionality) offering opportunities to interoperate with both.
```

Comment:
The wording is a bit misleading I think. It is not obvious from the first reading, that both gRPC and REST integrations, are not supported yet.